dev/devicelab/lib/framework/running_processes.dart

```diff
@@ -88,8 +87,7 @@ Stream<RunningProcessInfo> windowsRunningProcesses(String processName) async* {
       ? '"Get-CimInstance Win32_Process -Filter \\"name=\'$processName\'\\" | Select-Object ProcessId,CreationDate,CommandLine | Format-Table -AutoSize | Out-String -Width 4096"'
       : '"Get-CimInstance Win32_Process | Select-Object ProcessId,CreationDate,CommandLine | Format-Table -AutoSize | Out-String -Width 4096"';
   // Unfortunately, there doesn't seem to be a good way to get ProcessManager to
-  // run this. May be a bug in Dart.
-  // TODO(dnfield): fix this when https://github.com/dart-lang/sdk/issues/36175 is resolved.
```

Comment:
The linked issue is resolved, but appears to be un-actionable.

Reply:
Yup, I was just wrong here.